### DISCUSSION TITLE
Query String parameters from URL

### DISCUSSION THREAD

**Discussion by kellis5137** [Budibase/budibase#5942] [Help]
Is there any way to get query string parameters from the URL? For example, if someone calls my app:

https://budibase.example.com/app/appname#/admin/test?param1=test1&param2=test2

Can I retrieve param1 and param2 similar to how we would retrieve URL parameters (example URL.param1)?

**Comment by melohagan** [comment]
Hi @kellis5137 

In budibase, we use URL variables separated by a forward slash: https://docs.budibase.com/docs/url-parameters#multiple-variables

If you really want to use URL parameters, this answer may help: https://github.com/Budibase/budibase/discussions/5157#discussioncomment-2468159
